lib/system/freertos/CMakeLists.txt

```diff
@@ -21,4 +21,12 @@ if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${PROJECT_MACHINE})
   add_subdirectory(${PROJECT_MACHINE})
 endif (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/${PROJECT_MACHINE})
 
+if ("${PROJECT_MACHINE}" STREQUAL "zynqmp_a53" OR
+    "${PROJECT_MACHINE}" STREQUAL "zynqmp_a72" OR
+    "${PROJECT_MACHINE}" STREQUAL "zynqmp_a78" OR
+    "${PROJECT_MACHINE}" STREQUAL "zynqmp_r5" OR
+    "${PROJECT_MACHINE}" STREQUAL "zynq7")
+add_subdirectory(xlnx_common)
```

Comment:
The folder name doesn't seem maintainable in long term.
what about Xilinx or AMD, then in case of new AMD platforms some subfolders could be added

Reply:
@arnopo it's not decided yet. For now I suggest we keep vendor name as xlnx. Also, Platform BSP API used for xlnx platforms and AMD platforms will be different, so we will need separate directory for them anyway.

Reply:
Actually I prefer we keep one directory only for Xilinx's platforms and second directory for AMD's platforms as they both will have separate BSPs and so API used to implement interfaces will be different too.

If we want to avoid xlnx vendor name, We can go ahead with "zynqmp_common" name, as it is "zynqmp" family platforms.

@arnopo are you ok with that?